src/api/v1/permissions.json

```diff
@@ -0,0 +1,116 @@
+    "user_details-child:read",
+    "users_details-child:write",
+    "user_details-own:write",
+    "user_details-own:read",
+    "user_details-own:delete",
+    "constants-all:write",
+    "constants-all:delete",
+    "visit_activies-own:write",
+    "visit_activies-own:delete",
+    "organisations_feedback-parent:write",
+    "visit_logs-parent:write",
+    "volunteer_logs-parent:write",
+    "volunteer_logs-parent:delete"
+  ],
+  "permissionsForRoles": {
+    "TWINE_ADMIN": [
+      "constants-all:read",
+      "organisations_details-child:read",
+      "organisations_details-child:write",
+      "organisations_details-child:delete",
+      "organisations_subscriptions-child:read",
+      "organisations_training-child:read",
+      "organisations_training-own:delete",
+      "visit_activies-own:read",
+      "organisations_feedback-child:read",
+      "users_details-child:read",
+      "visit_logs-child:read",
+      "volunteer_logs-child:read",
+      "organisations_outreach-child:read",
+      "user_details-child:read",
+      "users_details-child:write",
+      "user_details-own:write",
+      "user_details-own:read",
+      "user_details-own:delete",
+      "constants-all:write",
+      "constants-all:delete"
+    ],
+    "FUNDING_BODY": [
+      "constants-all:read",
+      "organisations_details-child:read",
+      "organisations_details-child:write",
+      "organisations_details-child:delete",
+      "organisations_subscriptions-child:read",
+      "organisations_training-child:read",
+      "organisations_training-own:delete",
+      "visit_activies-own:read",
+      "organisations_feedback-child:read",
+      "visit_logs-child:read",
+      "volunteer_logs-child:read",
+      "organisations_outreach-child:read",
+      "users_details-child:read",
+      "users_details-child:write",
+      "user_details-own:write",
+      "user_details-own:read",
+      "user_details-own:delete" 
+    ],
+    "ORG_ADMIN": [
+      "constants-all:read",
+      "organisations_details-child:read",
+      "organisations_details-child:write",
+      "organisations_details-child:delete",
+      "organisations_training-child:read",
+      "organisations_training-own:delete",
+      "visit_activies-own:read",
+      "visit_activies-own:write",
+      "visit_activies-own:delete",
+      "organisations_feedback-child:read",
+      "organisations_feedback-parent:write",
+      "users_details-child:read",
+      "visit_logs-child:read",
+      "volunteer_logs-child:read",
+      "organisations_outreach-child:read",
+      "users_details-child:write",
+      "user_details-own:write",
+      "user_details-own:read",
+      "user_details-own:delete" 
+    ],
+    "ORG_NON_ADMIN": [],
+    "VISITOR": [
+      "constants-all:read",
+      "visit_activies-own:read",
+      "visit_logs-parent:write",
+      "user_details-own:write",
+      "user_details-own:read" 
+    ],
+    "VOLUNTEER": [
+      "constants-all:read",
+      "volunteer_logs-parent:write",
```

Comment:
Would `volunteer_logs-own:...` make more sense here? Do volunteers "own" the logs they create? Or is it the CBs?